.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,38 @@
+name: Integration Tests
```

Comment:
Do you intend to add a new workflow? 
Maybe this is meant to be "moved" to the original uniswap dire? (git mv?)

Reply:
Yep.

I've only merged and fixed conflicts, did not stop to adjust all the changes yet